### DISCUSSION TITLE
Unexpected error occurred and unable to upload contents

### DISCUSSION THREAD

**Discussion by Maurizio-Bianchi** [requarks/wiki#6977] [Help / Questions]
After having installed successfully and smoothly WikiJS, in /var/wiki the system finalized the setup and created all the database structures. No error during setup, everything seems to work fine. Webserver is accessible but everytime I try to upload an image or change the logo I get a red banner "An unexpected error occurred." disappearing after a while and no content can be uploaded.
No error in logs. Any clue?

OS: AlmaLinux 8.9 and NodeJS v14.21.3
Wiki.js version: 2.5.301
Database engine: MySQL Percona 5.7`

**Comment by fnafox** [comment]
Does the user your wiki process is running under have write access to `/var/wiki`?
